lib/rake/build.rake

```diff
@@ -81,7 +81,7 @@ namespace :build do
         else
           ChatClient.log 'Seeding <b>dashboard</b>...'
           ChatClient.log 'consider setting "skip_seed_all" in locals.yml if this is taking too long' if rack_env?(:development)
-          RakeUtils.rake 'seed:all'
+          RakeUtils.rake 'seed:all', (rack_env?(:test) ? '--trace' : nil)
```

Comment:
should I include `ENV['CI']` as well?  My understanding is that would detect the CircleCI environments

Reply:
`RACK_ENV` is `test` in our CircleCI builds as well: https://github.com/code-dot-org/code-dot-org/blob/cf86dfa5e97a1ff3c7f217216df38654d84d4c75/.circleci/config.yml#L14-L16

Reply:
thanks!